sklearn/metrics/_classification.py

```diff
@@ -2486,6 +2486,6 @@ def brier_score_loss(y_true, y_prob, sample_weight=None, pos_label=None):
                 np.array_equal(labels, [-1])):
             pos_label = 1
         else:
-            pos_label = y_true.max()
+            pos_label = labels[-1]
```

Comment:
We need to update comment on the top of the if statemtent.
We need also to improve the documentation to be more specific in the docstring.

Reply:
the comment is still valid and we already noted down the expected behaviour: Defaults to the greater label unless y_true is all 0 or all -1 in which case pos_label defaults to 1.

Reply:
What I meant is to move this comment in the first branch (`if np.array_equal ...`) of the if and add the behavior in the second one.

Reply:
I think the comment is OK since "otherwise pos_label is set to the greater label" is still true here . But yeah the docstring could be clarified to say that strings are also ordered by lexicographic order (which makes absolutely no sense to me)